drivers/filters/fltmgr/Messaging.c

```diff
@@ -700,9 +700,10 @@ CreateClientPort(_In_ PFILE_OBJECT FileObject,
     }
 
     /* Now insert the new client port into the object manager*/
-    Status = ObInsertObject(ClientPortObject, 0, FLT_PORT_ALL_ACCESS, 1, 0, (PHANDLE)&PortHandle);
+    Status = ObInsertObject(ClientPortObject, NULL, FLT_PORT_ALL_ACCESS, 1, NULL, (PHANDLE)&PortHandle);
     if (!NT_SUCCESS(Status))
     {
+        ClientPortObject = NULL;
```

Comment:
maybe a comment about why this is nulled-out could be welcomed. For example:
`/* The function failed and already dereferenced ClientPortObject */`

Reply:
Added.